compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -786,6 +795,7 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             (args, None)
         };
 
+        let mut argument_tmps = vec![];
```

Comment:
nit: I think it would be nice if this had a slightly more descriptive name.

```suggestion
        let mut copied_constant_arguments = vec![];
```

Reply:
Adopted the suggestion